modules/vault-cluster/main.tf

```diff
@@ -29,10 +66,11 @@ resource "aws_autoscaling_group" "autoscaling_group" {
   wait_for_capacity_timeout = "${var.wait_for_capacity_timeout}"
 
   tags = ["${concat(
+    var.cluster_extra_tags,
```

Comment:
Why reverse the order of the tags?

Reply:
I don't want var.cluster_extra_tagsto inadvertently change the value of the Name tag.